### PR TITLE
Zoom in and out PDF

### PR DESCRIPTION
**Pull request by sarinali**
# Description

Ensures that zooming in and out works on PDF viewer

## Bugzilla bug ID

**Testrail:** https://mozilla.testrail.io/index.php?/cases/view/3928
**Link:** https://bugzilla.mozilla.org/show_bug.cgi?id=1910702

## Type of change

Please delete options that are not relevant.

- [x] New Test

# How does this resolve / make progress on that bug?

Finishes

# Screenshots / Explanations

N/A

# Comments / Concerns

N/A

### CODE REVIEW

File: modules/data/pdf_viewer.components.json [hunk 1]
```diff
@@ -1 +1,13 @@
-{}
\ No newline at end of file
+{
```
Comment (ben-c-at-moz) [inline modules/data/pdf_viewer.components.json:1]: Is the pdf viewer part of the chrome context? If so, you should add `"context": "chrome"`. If not, this should probably be part of `Generic_Pdf` (potentially with a rename).

**Review by ben-c-at-moz — COMMENTED** [review]
Just want some clarity on a somewhat important piece, but the code looks good.

File: modules/data/pdf_viewer.components.json (hunk 1 above)
Comment (sarinali) [reply modules/data/pdf_viewer.components.json]: fixed

File: tests/pdf_viewer/test_zoom_pdf_viewer.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        )
+    else:
+        pdf_page.zoom_in_toolbar()
+        pdf_page.wait.until(
+            lambda _: float(body.value_of_css_property("--scale-factor"))
+            > before_scale_factor
+        )
+
+
+# @pytest.mark.parametrize("control", controls)
```
Comment (ben-c-at-moz) [inline tests/pdf_viewer/test_zoom_pdf_viewer.py:33]: Is there a reason for all this commented code? Do you want it in the repo?

**Review by ben-c-at-moz — APPROVED** [review]
good to merge!

**Pull request merged**